### PR TITLE
add ogv circulating supply endpoint

### PR DESCRIPTION
**Pull request by rolandpo**
add circulating supply endpoint for OGV (small chance of this not working right away)

### CODE REVIEW

File: logic/scripts/token_stats.py [hunk 1]
```diff
@@ -239,11 +246,96 @@ def fetch_ogn_stats(ogn_usd_price,staked_user_count,staked_token_count,ogn_stake
 
     return out_data
 
-def update_circulating_supply(circulating_supply):
+def fetch_ogv_stats(ogv_usd_price):
+    total_supply = 4071219784
```
Comment (micahalcorn) [inline logic/scripts/token_stats.py:250]: Should this be hard-coded or pulled from elsewhere in this script?

File: logic/scripts/update_token_insight.py [hunk 2]
```diff
@@ -103,6 +103,8 @@ def call_amberdata(url):
 def call_ethplorer(url):
     url = "%s?apiKey=%s" % (url, constants.ETHPLORER_KEY or 'freekey')
     raw_json = requests.get(url)
+    print('AAAAAAAAAA')
```
Comment (micahalcorn) [inline logic/scripts/update_token_insight.py:106]: Should this be here?

**Review by micahalcorn — CHANGES_REQUESTED** [review]

File: logic/scripts/update_token_insight.py (hunk 2 above)
Comment (rolandpo) [reply logic/scripts/update_token_insight.py:106]: probably not

File: logic/scripts/token_stats.py (hunk 1 above)
Comment (rolandpo) [reply logic/scripts/token_stats.py:250]: this was hardcoded for OGN so just did the same here, can change it later maybe once this goes through

**Comment by rolandpo** [comment]
removed those lines, I would look at changing the OGN/OGV total supply stuff later on if that works

**Comment by rolandpo** [comment]
made some changes to the total supply functions

File: database/db_models.py [hunk 3]
```diff
@@ -151,8 +152,18 @@ class TokenTransaction(db.Model):
     def __str__(self):
         return '%s' % (self.name)
 
-class CirculatingSupply(db.Model):
-    __tablename__ = 'circulating_supply'
+class CirculatingOgnSupply(db.Model):
```
Comment (franckc) [inline database/db_models.py:155]: A migration file needs to be added in order for the new table to get created.
See https://github.com/OriginProtocol/origin-website/tree/master/migrations

See flask-migrate doc regrding how to generate a new migration file
https://flask-migrate.readthedocs.io/en/latest/

The migration runs automatically as part of the deploy
See https://github.com/OriginProtocol/origin-website/blob/087dc8aaa287f247d764f8544a5f69aa22163a9b/Procfile#L3

**Review by franckc — CHANGES_REQUESTED** [review]

File: database/db_models.py [hunk 4]
```diff
@@ -151,8 +152,18 @@ class TokenTransaction(db.Model):
...
+    id = db.Column(db.Integer, primary_key=True)
+    supply_amount = db.Column(db.Float(), default=0)
+    snapshot_date = db.Column(db.DateTime(timezone=True), server_default=db.func.now())
+    created_at = db.Column(db.DateTime(timezone=True), server_default=db.func.now())
+
+    def __str__(self):
+        return '%s' % (self.name)
+
+class CirculatingOgvSupply(db.Model):
+    __tablename__ = 'circulating_ogv_supply'
```
Comment (franckc) [inline database/db_models.py:166]: A migration file is also required to change that table name.

**Review by franckc — CHANGES_REQUESTED** [review]

**Comment by rolandpo** [comment]
added the migration file

File: migrations/versions/241461fb2314_.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+# revision identifiers, used by Alembic.
+revision = '241461fb2314'
+down_revision = '3fae31bbf999'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('circulating_ogn_supply',
```
Comment (franckc) [inline migrations/versions/241461fb2314_.py:21]: As opposed to creating a new table with no historical data which I supposed will cause issues on the front-end, how about renaming the table from `circulating_supply` to `circulating_ogn_supply`?

Comment (rolandpo) [reply migrations/versions/241461fb2314_.py:21]: ah right, I’ll have a look

**Comment by rolandpo** [comment]
ok replaced the migration file, I think it makes more sense now

**Review by franckc — APPROVED** [review]
LGTM!

**Pull request merged**